src/common/xoa-updater.js

```diff
@@ -208,7 +222,7 @@ class XoaUpdater extends EventEmitter {
       return c
     } else {
       return eventToPromise.multi(c, ['open'], ['closed', 'error'])
-      .then(() => handleOpen(c), openFailure)
+      .return(c)
```

Comment:
Please, use standard API promise, there are no guarantee that this code will always use Bluebird.